lib/languages.ts

```diff
@@ -648,6 +648,17 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    v: {
+        name: 'V',
+        monaco: 'v',
+        extensions: ['.v'],
```

Comment:
You might also want to add `.vsh`

Reply:
fixed in [627bdfb](https://github.com/compiler-explorer/compiler-explorer/pull/5297/commits/627bdfb5f0377a4e6164f791f640ac8568be2aed)